### ISSUE TITLE
Possibly some common issues.

### ISSUE THREAD

**Issue by eggcrab** [KillzXGaming/Switch-Toolbox#191]
I'm sort of new to this whole Github thing but I thought i'd try downloading this to extract some game models and such, so sorry if i sound stupid. One of the problems I experienced right off was this:
![2020-02-10 (2)](https://user-images.githubusercontent.com/60910142/74211854-f2a0b580-4c4e-11ea-9c8a-8f6e1e0950b3.png)
which I believe is a missing dll, but I have no idea why its missing or how to redownload it and where. This was after i just downloaded and extracted the zip file and ran the .exe.

Another issue I had (which hopefully wasn't caused by the problem above) was 
![2020-02-10 (3)](https://user-images.githubusercontent.com/60910142/74212327-e289d580-4c50-11ea-8489-48fa9d7fb9bf.png)
Im trying to get the models from Luigi's Mansion 3, and i tried to open a file called global.dict which contains a lot of models, and it gave me this error message for quite a while until it started magically working again, and then gave me the error message again when i tried to open another model inside of the global.dict file.

I was also wondering if there was bone/rigs available in .dae files, since thats the only way i can export these models. Im looking to open them in blender and have the weights and all of that there, if i can find it, but it doesn't seem to show up in the model preview or the .dae files. Does there need to be a new update to support files like the LM3 files? Im not sure, sorry though, I really don't have that much knowledge of this. I was just wondering, since this seems to be the best option there is out there for exploring switch files.

**Comment by KillzXGaming** [comment]
Make sure you right click the zip and check unblock. Then extract the files. Also make sure this is installed https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads

**Comment by KillzXGaming** [comment]
This should be working fine now but if you still have issues feel free to reopen this.
